includes/class-ssp-frontend.php

```diff
@@ -748,22 +819,308 @@ public function media_player ( $src = '', $episode_id = 0 ) {
+                                        [
+                                            'trackEvent', 'Player', 'Back 30 Seconds', 'Global Stats'
+                                        ]
+                                    );*/
+
+                                    window.ssp_player<?php echo $largePlayerInstanceNumber; ?>.skipBackward();
+
+                                } );
+
+                                $('#ssp_player_id_<?php echo $largePlayerInstanceNumber; ?> #ssp_playback_speed<?php echo $largePlayerInstanceNumber; ?>').on( 'click', function(e){
+                                    switch( $( e.currentTarget ).parent().find( '[data-ssp-playback-rate]' ).attr( 'data-ssp-playback-rate' ) ){
+                                        case "1":
+                                            $( e.currentTarget ).parent().find( '[data-ssp-playback-rate]' ).attr( 'data-ssp-playback-rate', '1.5' );
+                                            $( e.currentTarget ).parent().find( '[data-ssp-playback-rate]' ).text('1.5X' );
+                                            window.ssp_player<?php echo $largePlayerInstanceNumber; ?>.setPlaybackRate(1.5);
+                                            break;
+                                        case "1.5":
+                                            $( e.currentTarget ).parent().find( '[data-ssp-playback-rate]' ).attr( 'data-ssp-playback-rate', '2' );
+                                            $( e.currentTarget ).parent().find( '[data-ssp-playback-rate]' ).text('2X' );
+                                            window.ssp_player<?php echo $largePlayerInstanceNumber; ?>.setPlaybackRate(2);
+                                            break;
+                                        case "2":
+                                            $( e.currentTarget ).parent().find( '[data-ssp-playback-rate]' ).attr( 'data-ssp-playback-rate', '1' );
+                                            $( e.currentTarget ).parent().find( '[data-ssp-playback-rate]' ).text('1X' );
+                                            window.ssp_player<?php echo $largePlayerInstanceNumber; ?>.setPlaybackRate(1);
+                                        default:
+                                            break;
+                                    }
+                                } );
+
+                            } );
+
+                        </script>
+
+                        <?php
+
+                        $player = ob_get_clean();
+
+                        // ---- /NEW PLAYER -----
+                    }
+
+				    break;
+
+			        case 'video':
+
 					// Use featured image as video poster
 					if( $episode_id && has_post_thumbnail( $episode_id ) ) {
 						$poster = wp_get_attachment_url( get_post_thumbnail_id( $episode_id ) );
 						if( $poster ) {
 							$params['poster'] = $poster;
 						}
 					}
+
 					$player = wp_video_shortcode( $params );
 				break;
 			}
```

Comment:
Not sure if this is a new bug, but on viewing my podcast archive I get the following error 
Getting the following error when I view my podcast archive.

Notice: Undefined variable: src in /var/www/public/wp-content/plugins/seriously-simple-podcasting/includes/class-ssp-frontend.php on line 1129

Reply:
@jonathan - If I look on said line, I only see this code: 

`$attachment_id = absint( attachment_url_to_postid( $url ) );`

I can see no relation to a `$src` variable. Can you please paste a screenshot so I can better see where the error originates? Are you referring to a series archive? If so, mine renders 100% so I need to see a little more :) thank you!

Reply:
@jonathanbossenger I found out that issue, I had renamed the episode source file variable from `$src` to `$srcFile` as `$src` was used in a php `list()` function for album art related stuff. Should be fixed.